awgportal.Dockerfile

```diff
@@ -12,8 +12,8 @@ ENV REACT_APP_WEBSITE_NAME=GDC \
     REACT_APP_GDC_AUTH_API="https://portal.awg.gdc.cancer.gov/auth/api" \
     REACT_APP_AWG=true \
     REACT_APP_IS_AUTH_PORTAL=true \
-    # REACT_APP_GDC_DISPLAY_SLIDES=true \
-    REACT_APP_SLIDE_IMAGE_ENDPOINT="https://api.gdc.cancer.gov/tile/" \
+    REACT_APP_GDC_DISPLAY_SLIDES=true \
```

Comment:
you don't want to enable this flag by default until the feature is ready to release, as that'd make it visible by the public.

Reply:
Acknowledged.